### PR TITLE
Minor cleanup, no functional changes intended.

### PR DESCRIPTION
**Pull request by meisterT**

### CODE REVIEW

File: webapp/tests/E2E/Controller/ControllerRolesTraversalTest.php [hunk 1]
```diff
@@ -152,8 +152,9 @@ protected function verifyAccess(array $combinations, array $roleURLs) : void
      * Test that having the team role for example is enough to view pages of that role.
      * This test should detect mistakes where a page is disabled when the user has a
      * certain role instead of allowing when the correct role is there.
-     * @var string[] $baseRoles The standard role of the user
-     * @var string[] $optionalRoles The roles which should not restrict the viewable pages
+     * @var string $roleBaseURL The base URL of the role.
```
Comment (mvr320) [inline webapp/tests/E2E/Controller/ControllerRolesTraversalTest.php:155]: 1 extra space maybe?

File: webapp/tests/Unit/Controller/API/BaseTest.php [hunk 2]
```diff
@@ -206,12 +206,12 @@ public function testList()
     }
 
     /**
-     * Test that the list action returns the expected objects if ID's are passed
+     * Test that the list action returns the expected objects if IDs are passed..
```
Comment (mvr320) [inline webapp/tests/Unit/Controller/API/BaseTest.php:209]: .. -> .
.. -> ...

File: webapp/tests/Unit/Controller/API/GeneralInfoControllerTest.php [hunk 3]
```diff
@@ -160,7 +160,7 @@ public function testCountryFlagNotFound(string $countryCode, string $size)
             /** @var BinaryFileResponse $response */
             $response = $this->client->getResponse();
             static::assertEquals(404, $response->getStatusCode());
-            static::assertEquals(sprintf('country flag for %s of size %s not found', strtoupper($countryCode), $size), json_decode($response->getContent(), true)['message']);
+            static::assertEquals(sprintf('Country flag for %s of size %s not found.', strtoupper($countryCode), $size), json_decode($response->getContent(), true)['message']);
```
Comment (mvr320) [inline webapp/tests/Unit/Controller/API/GeneralInfoControllerTest.php:163]: Should you not also update the API which sets this value?

File: webapp/tests/Unit/Controller/API/LanguageControllerTest.php [hunk 4]
```diff
@@ -23,7 +23,7 @@ class LanguageControllerTest extends BaseTest
         ],
     ];
 
-    // Kotlin has allow_submit=false by default, so we don't expect it.
+    // Kotlin has allow_submit=false by default, so we don't expect it..
```
Comment (mvr320) [inline webapp/tests/Unit/Controller/API/LanguageControllerTest.php:26]: why double `.`?

File: webapp/tests/Unit/Controller/API/JudgehostControllerTest.php [hunk 5]
```diff
@@ -44,14 +44,14 @@ public function provideSingle(): Generator
...
+     * Test that the endpoint returns an empty list for objects that don't exist.
      *
      * @dataProvider provideSingleNotFound
      */
     public function testSingleNotFound(string $id): void {
         $id = $this->resolveReference($id);
         $url = $this->helperGetEndpointURL($this->apiEndpoint, $id);
         $object = $this->verifyApiJsonResponse('GET', $url, 200, $this->apiUser);
-        static::assertEquals($object, []);
+        static::assertEquals([], $object);
```
Comment (mvr320) [inline webapp/tests/Unit/Controller/API/JudgehostControllerTest.php:55]: To me it reads nicer that object (the thing you worked on) is equal to something (the empty list), but this is probably convention?

File: webapp/tests/E2E/Controller/ControllerRolesTraversalTest.php (hunk 1 above)
Comment (meisterT) [reply webapp/tests/E2E/Controller/ControllerRolesTraversalTest.php]: Done

File: webapp/tests/Unit/Controller/API/BaseTest.php (hunk 2 above)
Comment (meisterT) [reply webapp/tests/Unit/Controller/API/BaseTest.php]: Done

File: webapp/tests/Unit/Controller/API/GeneralInfoControllerTest.php (hunk 3 above)
Comment (meisterT) [reply webapp/tests/Unit/Controller/API/GeneralInfoControllerTest.php]: I didn't want to include this in this PR.

File: webapp/tests/Unit/Controller/API/LanguageControllerTest.php (hunk 4 above)
Comment (meisterT) [reply webapp/tests/Unit/Controller/API/LanguageControllerTest.php]: Fixed.

File: webapp/tests/Unit/Controller/API/JudgehostControllerTest.php (hunk 5 above)
Comment (meisterT) [reply webapp/tests/Unit/Controller/API/JudgehostControllerTest.php:55]: indeed, it's convention: https://phpunit.de/manual/6.5/en/appendixes.assertions.html#appendixes.assertions.assertEquals

**Review by eldering — APPROVED** [review]
LGTM, didn't check in full detail.

**Pull request merged**